src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -101,6 +101,11 @@ public async Task<IActionResult> WithTransferSender(CohortsByProviderRequest req
         [Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
         public async Task<IActionResult> AddDraftApprenticeship(CreateCohortWithDraftApprenticeshipRequest request)
         {
+            if (_featureTogglesService.GetFeatureToggle(ProviderFeature.DeliveryModel).IsEnabled)
+            {
+                return RedirectToAction(nameof(SelectCourse), request);
+            }
+
```

Comment:
Looking into this the service which should be used is SFA.DAS.Authorization.Services.IAuthorizationService as the IFeatureToggle.... is more specific to the Provider context. I think the use of this service in the cohort page was incorrect.